css/style.css

```diff
@@ -0,0 +1,84 @@
+header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  height: 48px;
+  width: 100%;
+  margin: 0;
+}
+
+.logo-container {
+  box-sizing: border-box;
+  height: 48px;
+  padding: 10px 12px 10px 12px;
+  display: flex;
+}
+
+.text-logo {
+  color: #28352f;
+  font-family: 'Inter', sans-serif;
+  font-weight: 600;
+  font-size: 20px;
+  text-decoration: none;
+}
+
+.nav-menu {
+  width: 40px;
+  height: 40px;
+  right: 24px;
+  display: flex;
+  align-items: center;
+}
+
+.headline {
+  background-image: url(/images/header-illsutration-mobile.svg);

```

Comment:
- [ ] As you can see the background is not being found. This is for the same situation as the URL path for the other images. I would suggest looking for the correct path, here is a good example, your css file that is trying to look for the image is inside the `css/` folder, which means that you should go one directory back using `../` and from there go to the `images` folder.

![image](https://user-images.githubusercontent.com/81522435/138138475-c84009a2-c003-46a9-95c7-e4d0a0bdcf72.png)

- [OPTIONAL] Also I would suggest using single quotes when using a PATH in your CSS for better practices. This line should look like:
  - `background-image: url('../images/header-illsutration-mobile.svg');`